index.html

```diff
@@ -21,6 +21,10 @@ <h2>Store</h2>
 			</a>
 			<div class="news">
 				<h2>Latest News</h2>
+				<article>
+					<h3><a href="https://kafka.apache.org/downloads#2.3.1">AK Release 2.3.1</a></h3>
+					<time>October 24, 2019</time>
+				</article>
```

Comment:
@mumrah We should remove the entry about `2.3.0` release

Reply:
I wasn't sure about that since these `<article>` seem pretty new. I'll remove the 2.3.0 section